src/operator/nn/ctc_loss.cu

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * Copyright (c) 2018 by Contributors
+ * \file ctc_loss.cu
+ * \brief GPU Implementation of ctc_loss op
+ */
+
+#include "./ctc_loss-inl.h"
+#include "../contrib/ctc_include/detail/gpu_ctc.h"
+
+namespace mshadow {
+
```

Comment:
same comment as above, would it be simpler to just put "using namespace mshadow;" in the body of the function?

Reply:
namespace is needed here because compute_ctc_loss function is being called inside ctc_loss-inl.h under namespace mshadow.